ch_pipeline/analysis/flagging.py

```diff
@@ -1246,29 +1309,35 @@ def daytime_flag(time):
     return flag
 
 
-def transit_flag(body, time, nsigma=2.0):
+def transit_flag(body, time, freq=400.0, nsigma=2.0):
```

Comment:
`freq` argument should probably be after `nsigma` so as not to break any existing calls to this function which provide `nsigma` as a positional argument rather than a keyword

Reply:
Good catch, thanks!